tests/ga/test_extension.py

```diff
@@ -1620,13 +1620,41 @@ def test_ext_path_and_version_env_variables_set_for_ever_operation(self, *args):
 
             # Extension Path and Version should be set for all launch_command calls
             for args, kwargs in patch_start_cmd.call_args_list:
-                self.assertIn(EXTENSION_PATH, kwargs['env'])
-                self.assertIn('OSTCExtensions.ExampleHandlerLinux-1.0.0', kwargs['env'][EXTENSION_PATH])
-                self.assertIn(EXTENSION_VERSION, kwargs['env'])
-                self.assertEqual("1.0.0", kwargs['env'][EXTENSION_VERSION])
+                self.assertIn(ExtCommandEnvVariable.ExtensionPath, kwargs['env'])
+                self.assertIn('OSTCExtensions.ExampleHandlerLinux-1.0.0', kwargs['env'][ExtCommandEnvVariable.ExtensionPath])
+                self.assertIn(ExtCommandEnvVariable.ExtensionVersion, kwargs['env'])
+                self.assertEqual("1.0.0", kwargs['env'][ExtCommandEnvVariable.ExtensionVersion])
 
         self._assert_handler_status(protocol.report_vm_status, "Ready", expected_ext_count=1, version="1.0.0")
 
+    def test_ext_sequence_no_should_set_for_ever_command_call(self, *args):
```

Comment:
Typo `ever` -> `every`? Also should it be `sequence_no_should_**be**_set` instead? Maybe `updated` bears more meaning since it's always set as an incrementally increasing value?

Reply:
Thanks for the typo catch! We're not updating the seq no, just passing what we got in the goalstate to the extension commands